app-backend/database/src/main/scala/com/azavea/rf/database/tables/Organizations.scala

```diff
@@ -60,6 +61,9 @@ object Organizations extends TableQuery(tag => new Organizations(tag)) with Lazy
     }
   }
 
+  def listOrganizations = pageOrganizations(Organizations)
+  def listOrganizations(ids: Seq[UUID]) = pageOrganizations(Organizations.filter(_.id inSet ids))
```

Comment:
I've read that it is best to avoid method overloading in Scala, though I feel like I don't have a strong enough grounding in the language yet to have an opinion about whether this is a valid use case or what the best alternative would be here. This seems like a reasonable solution but it might be good to look into whether there's an alternative to overloading here (though if this turns into a rabbit-hole I think bailing out is perfectly fine).